src/Online_Bookstore/Dashboard/BookSearch.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+import { Form } from "react-bootstrap";
+import useFetch from "../CustomHooks/useFetch";
+import BooksView from "./BooksView";
+
+
+const BookSearch = (props) => {
+  const [search, setSearch] = useState("");
+  const [searchedBook, setSearchedBook] = useState([]);
```

Comment:
same as above, combine into single state

@Poorvi-3pillar

Reply:
ma'am in this case both the states are used for different purposes